src/taglibs/migrate/layout-put-tag.js

```diff
@@ -0,0 +1,35 @@
+const commonTagMigrator = require("./all-tags");
+
+module.exports = function migrator(oldNode, context) {
+    const attributes = oldNode.attributes;
+    commonTagMigrator(oldNode, context);
+    oldNode.setTransformerApplied(commonTagMigrator);
+
+    context.deprecate(
+        'The "<layout-put>" tag is deprecated. Please use "<@tags>" instead. See: https://github.com/marko-js/marko/wiki/Deprecation:-var-assign-invoke-tags'
```

Comment:
We'll need a wiki page for this. I'm thinking it'd make sense to just have a shared page for all `layout` tags.

Reply:
Cool. I will create a wiki page for `layout` and update it here